src/main/java/club/issizler/okyanus/api/json/ClickFeature.java

```diff
@@ -0,0 +1,19 @@
+package club.issizler.okyanus.api.json;
+
+import org.jetbrains.annotations.NotNull;
+
+public class ClickFeature implements JsonFeature {
+
+    private final JsonEvent jsonEvent;
+
+    public ClickFeature(@NotNull final JsonEvent jsonEvent) {
+        this.jsonEvent = jsonEvent;
+    }
+
+    @NotNull
+    @Override
+    public String handle() {
+        return ",\"clickEvent\":{\"action\":\"" + jsonEvent.action() + "\",\"value\":" + jsonEvent.value() + '}';
```

Comment:
Honestly, you might want to use a proper JSON serialization library, like Gson (which should be included with Minecraft (or Fabric, can't remember), and thus free to use for us)

Reply:
I don't know how to use it. For now, it's working, maybe later I can improve it with the Gson.